bin/img2aspect.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# img2aspect
```

Comment:
This has been in my personal `bin` for a long time - I noticed that all of our blog feature images are being rendered differently, so I added the script to our `bin` so we could use it to normalize the images if we wanted.